ccip-sdk/src/ton/index.integration.test.ts

```diff
@@ -0,0 +1,778 @@
+import assert from 'node:assert/strict'
+import { before, describe, it } from 'node:test'
+
+import { Address } from '@ton/core'
+
+import { TONChain } from './index.ts'
+import type { CCIPMessage_V1_6_TON } from './types.ts'
+
+describe('TON index integration tests', () => {
```

Comment:
requires a real connection but I think it's a nice safety net, lmk what do you think